pkg/ingress/ingresstranslation/ingresstranslation.go

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ingresstranslation
+
+import (
+	"sync"
+
+	"istio.io/istio/pilot/pkg/model"
+	"istio.io/istio/pkg/config"
+	"istio.io/istio/pkg/config/schema/collection"
+	"istio.io/istio/pkg/config/schema/gvk"
+	"k8s.io/client-go/tools/cache"
+
+	ingressconfig "github.com/alibaba/higress/pkg/ingress/ingresstranslation/config"
+	"github.com/alibaba/higress/pkg/ingress/kube/common"
+	. "github.com/alibaba/higress/pkg/ingress/log"
+	"github.com/alibaba/higress/pkg/kube"
+)
+
+var (
+	_ model.ConfigStoreCache = &IngressTranslation{}
+	_ model.IngressStore     = &IngressTranslation{}
+)
+
+type IngressTranslation struct {
+	ingressconfig      *ingressconfig.IngressConfig
+	kingressconfig     *ingressconfig.KIngressConfig
```

Comment:
Letter `c` should be in its uppercase for variables names like `ingressconfig` and `kingressconfig`.